google/resource_compute_instance_group_migrate_test.go

```diff
@@ -8,24 +8,43 @@ import (
 
 func TestComputeInstanceGroupMigrateState(t *testing.T) {
 	cases := map[string]struct {
-		StateVersion int
-		Attributes   map[string]string
-		Expected     map[string]string
-		Meta         interface{}
+		StateVersion       int
+		Attributes         map[string]string
+		ExpectedAttributes map[string]string
+		ExpectedId         string
+		Meta               interface{}
 	}{
-		"change instances from list to set": {
+		"v1 to v2": {
```

Comment:
It doesn't look like this test is actually running:
```
=== RUN   TestComputeInstanceGroupMigrateState
2017/07/19 10:00:58 [INFO] Found Compute Instance Group State v0; migrating to v1
2017/07/19 10:00:58 [DEBUG] Attributes before migration: map[string]string{"instances.#":"1", "instances.0":"https://www.googleapis.com/compute/v1/projects/project_name/zones/zone_name/instances/instancegroup-test-1", "instances.1":"https://www.googleapis.com/compute/v1/projects/project_name/zones/zone_name/instances/instancegroup-test-0"}
2017/07/19 10:00:58 [DEBUG] Attributes after migration: map[string]string{"instances.#":"1", "instances.1519187872":"https://www.googleapis.com/compute/v1/projects/project_name/zones/zone_name/instances/instancegroup-test-0", "instances.764135222":"https://www.googleapis.com/compute/v1/projects/project_name/zones/zone_name/instances/instancegroup-test-1"}
--- PASS: TestComputeInstanceGroupMigrateState (0.00s)
```
It's not showing anything migrating to v2